app/models/state_file_base_intake.rb

```diff
@@ -304,4 +304,16 @@ def sanitize_bank_details
       self.date_electronic_withdrawal = nil
     end
   end
+
+  def calculate_age(inclusive_of_jan_1: true, dob: nil)
+    # federal guidelines: you qualify for age related benefits the day before your birthday
+    # that means for a given tax year those born on Jan 1st the following tax-year will be included
+    # this does not apply for benefits you age out of or any age calculations for Maryland
+    birth_year = dob.year
+    if inclusive_of_jan_1 && state_code != 'md'
```

Comment:
is there a way to avoid having the 'md' state_code here? should it be enough to use `inclusive_of_jan_1`?

Reply:
Why would we want to avoid having md state code in here? But yes we could have Maryland overwrite the function in their own state intake class @mpidcock

Reply:
Not using individual state codes in conditional logic was a principle we decided on after last tax season and the main reason we introduced the StateInformationService. It makes maintenance harder if State-specific rules are hidden in shared components making them harder to find, for example if we want to remove MD as a supported State. Overriding in the MD intake class would make the logic visible and isolated to the only State that is concerned with the exception.